cmake/tribits/core/installation/set_installed_group_and_permissions.cmake.in

```diff
@@ -107,3 +107,10 @@ ELSE()
     )
 
 ENDIF()
+
+# NOTE: Above, we had to change from using the var name 'CMAKE_INSTALL_PREFIX'
```

Comment:
Wow! This scoping is good to know about.

Reply:
CMake variable scoping rules are very strange.  If you are interested, I would recommend you get the book https://crascit.com/professional-cmake/ and then the chapter "Variables".